mhkit/tests/wave/io/test_wind_toolkit.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug 22 14:42:41 2022
+
+@author: akeeste
+"""
+
+from os.path import abspath, dirname, join, isfile, normpath, relpath
+from pandas.testing import assert_frame_equal
+from numpy.testing import assert_allclose
+from scipy.interpolate import interp1d
+from random import seed, randint
+import matplotlib.pylab as plt
+from datetime import datetime
+import xarray.testing as xrt
+import mhkit.wave.io.wind_toolkit as wtk
+from io import StringIO
+import pandas as pd
+import numpy as np
+import contextlib
+import unittest
+import netCDF4
+import inspect
+import pickle
+import time
+import json
+import sys
+import os
+
+
+testdir = dirname(abspath(__file__))
+datadir = normpath(join(testdir,'..','..','..','..','examples','data','wave','wind_toolkit'))
+
+
+class TestWINDToolkit(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+
+        self.my = pd.read_csv(join(datadir,'wtk_multiyear.csv'),
+            index_col = 'time_index',
+            names = ['time_index','pressure_200m_0'],
+            header = 0,
+            dtype = {'pressure_200m_0':'float32'})
+        self.my.index = pd.to_datetime(self.my.index)
+
+        self.ml = pd.read_csv(join(datadir,'wtk_multiloc.csv'),
+            index_col = 'time_index',
+            names = ['time_index','windspeed_10m_0','windspeed_10m_1'],
+            header = 0,
+            dtype = {'windspeed_10m_0':'float32',
+                     'windspeed_10m_1':'float32'})
+        self.ml.index = pd.to_datetime(self.ml.index)
```

Comment:
Adam here you explicitly state header names, index, and data types. It seems to me that the csv files have all this information and pandas will interpret almost all of it. What is your thought process around the specificity of the imports? Do you think it will prevent future issues? Do you think the specificity of the imports could cause us to make incorrect assumptions if the format changes in the future or if users are less specific in their imports?

Reply:
Hi @ssolson 

Overall I'm not tied strongly to these import parameters. The `pd.read_csv` calls works very similarly without them. I can remove them if you feel strongly. See below for specifics:

> What is your thought process around the specificity of the imports? 

I largely based this test's formatting off of [mhkit/tests/wave/io/test_hindcast.py](https://github.com/MHKiT-Software/MHKiT-Python/blob/Develop/mhkit/tests/wave/io/test_hindcast.py), which also specifies these input. If there is not a specific reason that test was done that way, I am fine changing this one. 

>Do you think it will prevent future issues? Do you think the specificity of the imports could cause us to make incorrect assumptions if the format changes in the future or if users are less specific in their imports?

These import parameters are defined so that `pd.read_csv` gives the same metadata that the MHKiT function does by default. Since this `read_csv` call is only used in the tests, it will not affect users' work. The dataset format appears to be static, but if it did change then it could break this test. On the other hand the test breaking would alert us that something in the WIND Toolkit formatting has changed and we might need to check our functionality

Reply:
Okay I see. Yeah let us keep it as is, I simply wanted to understand if there was an explicit reason for laying out the tests this way that I was overlooking. Thanks!